File: Multiples of 3 or 5/v1.py
def multiples_of(numbers: list, of: int):
    return [n for n in numbers if not n % of]


def solution(number: int):
    output = 0
    if number <= 3:
        return output
    output += 3
    if number <= 5:
        return output
    output += 5
    if number == 6:
        return output
    numbers = list(range(6, number))
    return output + sum(
        set(multiples_of(numbers, 3) + multiples_of(numbers, 5))
    )


if __name__ == "__main__":
    print(solution(int(input())))

File: Create Phone Number/v1.lua
return {
    create_phone_number = function(numbers)
        local output = string.format("(%d%d%d) %d%d%d-%d%d%d%d", numbers[1], numbers[2], numbers[3], numbers[4],
            numbers[5], numbers[6], numbers[7], numbers[8], numbers[9], numbers[10])
        do return output end
    end
};
